src/pages/SearchPage.vue

```diff
@@ -17,8 +15,11 @@ onMounted(() => {
 <template>
   <page-container>
     <div :class="$style.container">
-      <page-title>検索結果: {{ search }}</page-title>
-      <user-list :members="users" />
+      <page-title>検索結果: {{ query }}</page-title>
+      <suspense :key="query">
+        <user-list :query="query ?? ''" :search="query ?? ' '" />
```

Comment:
不必要なqueryを消してほしいと伝え忘れていたんですが、これもお願いします！
```suggestion
        <user-list :query="query ?? ''" />
```